### ISSUE TITLE
Initial value not loaded

### ISSUE THREAD

**Issue by mebibou** [alexcheng1982/angular-timezone-select#9]
If the `ng-model` is already set with a value, the component does not display it. For example, I have this in my html:

``` html
<select ng-model="timezone"></select>
```

And in my controller I have:

``` js
$scope.timezone = window.moment.tz.guess();
```

But the select box is still displaying no value. Also, not sure if it is related, but the cross to delete the current value appears and the placeholder does not, then on first click the dropdown will open, on second click on the cross the value will be erased and the placeholder will appear.

**Comment by alexcheng1982** [comment]
This issue is fixed in 1.1.3.

**Comment by mebibou** [comment]
Great I'll test this asap, thanks
